examples/encryption-consumer.js

```diff
@@ -32,7 +32,7 @@ const Pulsar = require('pulsar-client');
     subscription: 'sub1',
     subscriptionType: 'Shared',
     ackTimeoutMs: 10000,
-    privateKeyPath: "./certificate/private-key.client-rsa.pem"
+    privateKeyPath: "./test-test-conf/private-key.client-rsa.pem"
```

Comment:
This path is appears to be wrong. It should be `./tests/test-conf`.

BTW, you should not change JS code to adapt your test script. Instead, you should change your test script to adapt the existing code, which is a little different from Python client.